docs/by-example/sqlalchemy/getting-started.rst

```diff
@@ -1,6 +1,39 @@
-==================
-SQLAlchemy Support
-==================
+===========================
+SQLAlchemy: Getting started
+===========================
+
+This section of the documentation outlines how to use CrateDB's SQLAlchemy
+integration. It demonstrates both basic database operations (insert, select,
+delete), creating and dropping tables, running queries with aggregations,
+as well as the use of complex and geospatial data types.
```

Comment:
This sounds similar to the crud document. Would it make sense to merge the two into one? Or instead clarify the difference and link from one to the other.

Reply:
Thanks for your attention. You are right. I've tried to improve this now with 5dc0e0436.

The previous version was a bit of a potpourri of different topics. From that mix, two new documents have been created. One is about "advanced querying", including exercises about aggregations and full-text capabilities, and the other one exercises operations on the CrateDB container and geospatial special types concisely.

- [SQLAlchemy: Advanced querying](https://crate-python--464.org.readthedocs.build/en/464/by-example/sqlalchemy/advanced-querying.html)
- [SQLAlchemy: Working with special CrateDB types](https://crate-python--464.org.readthedocs.build/en/464/by-example/sqlalchemy/working-with-types.html)